src/appleseed/renderer/kernel/rendering/pixelrendererbase.cpp

```diff
@@ -41,6 +42,8 @@ using namespace foundation;
 
 namespace renderer
 {
+const uint8 InvalidSampleHint = 1;
```

Comment:
Move after the "PixelRendererBase class implementation" block comment since this is technically part of the implementation of `PixelRendererBase`.